quickwit-cli/src/generate_markdown.rs

```diff
@@ -127,7 +126,7 @@ fn markdown_for_subcommand(
             println!(
                 "`--{}` {}{}    ", // space is line break
                 arg.get_name(),
-                arg.get_about().unwrap_or_default(),
+                arg.get_help_heading().unwrap_or_default(),
```

Comment:
why this change?

Reply:
ok the api change is a bit misleading

Reply:
Yep, about/get_about vanished. Beta APIS are unstable indeed.

Versioned docs for reference :
old: https://docs.rs/clap/3.0.0-beta.5/clap/struct.Arg.html
new: https://docs.rs/clap/3.0.14/clap/struct.Arg.html